### ISSUE TITLE
Missing styling in csaf webview

### ISSUE THREAD

**Issue by koplas** [ISDuBA/ISDuBA#164]
The styling used in the upstream [csaf_webview](https://github.com/csaf-poc/csaf_webview/blob/0e584e5f989dd1f5b14ce033ae7ae68331708e3c/build/css/main.css) project was not integrated into ISDuBA. Because of this, all headings have the same font size, subpoints have the same styling as the parents and the tables are invisible.

My proposal would be to adopt the styling that is used in the upstream project.

@ThomasJunk @cintek

**Comment by ThomasJunk** [comment]
The reason behind this is best explained with a bit of project history:

- Before starting the webview project there was an agreement to a very minimal CSS framework. My first choice was [Chota](https://jenil.github.io/chota/) which was used throughout the implementation phase
- But as with every framework you start to deviate from the default behavior because your visual needs differ so I had a mix of Chota and custom CSS
- Later on we decided to drop Chota as a whole 
- The upstream CSS is 100% custom CSS

For this project we decided to use [flowbite](https://flowbite-svelte.com/). Therefore I transplanted the main code from the webview and scrubbed away the custom CSS in the hope we would use flowbite components. 

I would counter propose:

- Use flowbite components
- Use [Tailwind](https://tailwindcss.com/) classes (as Tailwind comes with flowbite)
